docs/monitoring/autodiscovery.md

```diff
@@ -0,0 +1,60 @@
+title: Sematext Autodiscovery Monitoring
+description: Sematext Autodiscovery Monitoring automatically scans for services that can be monitored using Sematext Agent and, depending on your configuration, automatically sets up monitoring agents. 
+
+## Autodiscovery Monitoring
+
+The centerpiece of Autodiscovery feature is Sematext Agent which scans for services that can be monitoring by Sematext supported integrations. It works in both bare-metal/virtual machine and in container environments. The data about such discovered services is sent to Sematext backend and displayed in UI under Discovery option in the menu on the left.
+
+<img class="content-modal-image" alt="Sematext Autodiscovery UI" src="../images/monitoring/autodiscovery-ui.png" title="Sematext Autodiscovery UI">
+
+
+Autodiscovered services can be monitored in two ways:
+- **Manually** - in case of bare-metal/virtual machine setups, you can use classic Agent installation instructions known from the past. In container environments you can just add `MONITORING_TOKEN` as environment variable to your container and Sematext Agent will automatically match it to the type of discovered service and set up monitoring agent container specifically for that service. In some cases you'll also have to provide credentials (like MySQL username/password) as environment variable or in case of Kubernetes workloads as a secret. Again, just follow the classic monitoring installation instructions in UI to set this up.
+- **Automatically** - this feature is available via UI when clicking on discovered service in Discovery view. In many cases Sematext Agent will be able to set up monitoring agent without requiring any changes from you and without restart of service you wish to monitor. As services appear over time, Sematext Agent automatically ensures new monitoring agents are created when the rule specified in the UI defines so. For each discovered service type you can enable or disable automatic monitoring at any point. For example, if Sematext Agent discovered your Elasticsearch cluster and you see it in Discovery UI, you can easily start its monitoring by a single click in the UI, no actions are required from you on Elasticsearch nodes. Similarly, if you decide to stop monitoring the cluster, single click in the UI will do the trick again. Automatic autodiscovery monitoring is supported both in bare-metal/virtual machine and container environments. 
```

Comment:
this feature is available via UI when clicking on discovered service in Discovery view.  => zap

In many cases Sematext Agent will be able to set up monitoring agent without requiring any changes from you and without restart of service you wish to monitor. => In many cases Sematext Agent will be able to start and stop monitoring discovered services without requiring any changes from you and without restarting of services you wish to monitor.

As services appear over time, Sematext Agent automatically ensures new monitoring agents are created when the rule specified in the UI defines so. => As services start, Sematext Agent automatically ensures they are monitored according to the rules you specified. 

by a single click => with a single click

UI, no actions are required => UI.  No actions are required

Automatic autodiscovery monitoring is supported both in bare-metal/virtual machine and container environments.  => zap.  Already stated further above.